calculator/main.py

```diff
@@ -26,7 +27,22 @@ def perform_operation(calc, operation, params):
     :param params: Tuple containing the list of nums to operate with.
                    ie: (1, 2, 3, 4.5, -2)
     """
-    pass
+   
+   
+    if not type(params) is tuple: 
```

Comment:
It's preferred to use `isinstance()` instead of `type` unless absolutely necessary as `isinstance` takes into account inheritance (this will become important as we get into object oriented programming). So here it would be preferable to use:
```
if not isinstance(params, tuple):
```